specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/examples/ContainerApps_ListBySubscription.json

```diff
@@ -78,7 +78,16 @@
                 },
                 "runtime": {
                   "java": {
-                    "enableMetrics": true
+                    "enableMetrics": true,
+                    "javaAgent": {
+                      "enabled": true,
+                      "logLevelSettings": [
```

Comment:
Since this more than logLevel , should we call it logSettings extendibility?

Reply:
Current design is aslo extendibility. When adding some new settings, the payload will looks like:

```json
{
    "enabled": true,
    "logLevelSettings": [
        {
            "logger": "org.springframework.boot",
            "level": "debug"
        }
    ],
    "logXXXSettings": {
        "sth": "sth"
    }
}
```